wagtail_transfer/field_adapters.py

```diff
@@ -213,8 +215,13 @@ def __init__(self, field):
         self.related_base_model = get_base_model(field.related_model)
         self.is_parental = isinstance(self.related_field, ParentalKey)
         self.is_followed = (get_base_model(self.field.model)._meta.label_lower, self.name) in FOLLOWED_REVERSE_RELATIONS
+        logger.debug(f"Checking field adaptor: {self.name}, {get_base_model(self.field.model)}', {get_base_model(field.related_model)}")
```

Comment:
There is probably a better place for this to print out, but I found it handy for understanding where I was at in an import and what relations were being considered

Reply:
I updated this behind a conditional to reduce some log noise, but open to either option or if there's a better place for this sort of log